fedimint-load-test-tool/src/main.rs

```diff
@@ -510,60 +502,65 @@ async fn test_download_config(
         .collect())
 }
 
-async fn _test_connect_std_client(
-    mut cfg: ClientConfig,
-    users: u16,
-    duration: Duration,
-    limit_endpoints: Option<usize>,
-    event_sender: mpsc::UnboundedSender<MetricEvent>,
-) -> anyhow::Result<Vec<BoxFuture<'static, anyhow::Result<()>>>> {
-    if let Some(limit_endpoints) = limit_endpoints {
-        cfg.api_endpoints = cfg
-            .api_endpoints
-            .into_iter()
-            .take(limit_endpoints)
-            .collect();
-        info!("Limiting endpoints to {:?}", cfg.api_endpoints);
-    }
-    let clients = (0..users)
-        .map(|_| async {
-            let mut tg = TaskGroup::new();
-            let client = build_client(&cfg, &mut tg, None).await?;
-            Ok::<_, anyhow::Error>(client)
-        })
-        .collect::<Vec<_>>();
-    let clients = futures::future::try_join_all(clients).await?;
-    info!("Keeping {users} clients connected for {duration:?}");
-    Ok(clients
-        .into_iter()
-        .map(|client| {
-            let event_sender = event_sender.clone();
-            let f: BoxFuture<_> = Box::pin(async move {
-                let initial_time = fedimint_core::time::now();
-                while initial_time.elapsed()? < duration {
-                    let m = fedimint_core::time::now();
-                    client.api().fetch_epoch_count().await?;
-                    event_sender.send(MetricEvent {
-                        name: "fetch_epoch_count".into(),
-                        duration: m.elapsed()?,
-                    })?;
-                    fedimint_core::task::sleep(Duration::from_secs(1)).await;
-                }
-                Ok(())
-            });
-            f
-        })
-        .collect())
-}
+// TODO: Rewrite this to use new client builder with_conncetion
+
+// async fn _test_connect_std_client(
```

Comment:
@douglaz , This PR moves config download into the client builder, so we might need a new api on the builder to limit modify config endpoints on the client. Should I be looking to make such a change or can I just consider this test fn a low pri given it was already _disabled

Reply:
Don't worry about it, just remove this code